src/main/java/essenmakanan/command/Command.java

```diff
@@ -14,5 +14,9 @@ public Command(boolean isExit) {
         ui = new Ui();
     }
 
-    public abstract void executeCommand(RecipeList recipes, IngredientList ingredients, String input);
+    public boolean isExit() {
+        return isExit;
+    }
+
```

Comment:
is the isExit attribute the best way to end a function? Should we just check if the command is "bye" instead?

Reply:
i feel like the isExit is an overly complicated way to check if we should end the program

Reply:
maybe can check if command is a bye command for the do while loop, but i like that you extracted out a bye command though, it gives consistency in our code